test/tc_selinux_base_usage_test.go

```diff
@@ -100,12 +100,6 @@ func (e *e2e) testCaseSelinuxBaseUsage(nodes []string) {
 
 	e.logf("removing workload")
 	e.kubectl("delete", "pod", "errorlogger")
-
-	e.logf("removing policy")
-	e.kubectl("delete", "selinuxprofile", "errorlogger")
-
-	e.logf("assert policy was removed")
-	e.assertSelinuxPolicyIsRemoved(nodes, rawPolicyName, maxNodeIterations, sleepBetweenIterations)
```

Comment:
This seems to block the fedora selinux e2e test and I have no idea how it is related to my code change. @jhrozek do you have any insight here?